src/site/content/en/blog/yahoo-japan-news-bfcache/index.md

```diff
@@ -0,0 +1,161 @@
+hero: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/iyoxhI76AJ7l8sH6OtUe.png
+thumbnail: image/jL3OLOhcWUQDnR4XjewLBx4e3PC3/stTQH6TS9oLBON1LvoKG.png
+alt: A hero image displaying the Yahoo JAPAN News logo at left. At right is a comparison of two filmstrips showing the benefits of bfcache (page loads in 0.3 seconds) versus without (page loads in 3.3 seconds).
+authors:
+  - jlwagner
+date: 2023-07-25
+tags:
+  - blog
+  - performance
+  - case-study
+---
+
+[Back/forward cache](/bfcache/) (or bfcache) is a browser optimization that enables instant back and forward navigation. It significantly improves the browsing experience for users, especially for websites that involve many back and forth navigations.
+
+Yahoo! JAPAN News, one of the most popular news platforms in Japan, drove a concerted effort to improve their bfcache hit-rate and saw significant user experience and business improvements as a result. Specifically, the results of the A/B test they conducted showed that pages that use bfcache had a **9% increase in ads revenue**.
+
+This case study will explain howYahoo! JAPAN News removed the blockers for bfcache, and how bfcache drastically improved the user experience.
+
+## Removing blockers for bfcache
+
+bfcache has been available since version M86 on Chrome, and is also available on all modern browsers. However, taking full advantage of bfcache requires removing potential blockers on one's website. Some major blockers that Yahoo! JAPAN News faced were:
+
+1. Removing `unload` handlers
+2. Removing the [`no-store` directive](https://developer.mozilla.org/docs/Web/HTTP/Headers/Cache-Control#no-store) from any set `Cache-control` headers
+
+Here's how Yahoo! JAPAN News has removed the blockers above:
+
+- **Unload Handlers:** Use the `pagehide` event instead of `unload` event, as the `unload` event is [very unreliable](/bfcache/#never-use-the-unload-event). Also, `permission-policy: unload` was [launched in version M115](https://chromestatus.com/feature/5579556305502208) so that the websites can reliably remove `unload` handlers for specific origins. Chrome is also planning to [gradually deprecate `unload` handlers](https://github.com/fergald/docs/blob/master/explainers/permissions-policy-deprecate-unload.md#logistics-of-deprecation).  
+  
+  You can check what major blockers there are for your website by going to **Chrome Dev Tools > Applications > Back/forward Cache** ([more details](/bfcache/#test-to-ensure-your-pages-are-cacheable)), or by calling [`notRestoredReasons` API](https://github.com/rubberyuzu/bfcache-not-retored-reason/blob/main/NotRestoredReason.md) for a more comprehensive view of blockers based on actual usage in the wild.
+- **`Cache-control: no-store`** (or CCNS for short): Changing the `Cache-control` header from **`no-store`** to **`no-cache`** can enable bfcache. Chrome is also planning to start [caching for bfcache even with a `no-store` header](https://chromestatus.com/feature/6705326844805120).
```

Comment:
```suggestion
- **`Cache-control: no-store`** (or CCNS for short): Changing the `Cache-control` header from **`no-store`** to **`no-cache`** can enable bfcache. Chrome is also planning to start [caching for bfcache even with a `no-store` header in certain circumstances](https://chromestatus.com/feature/6705326844805120).
```

I was hesitant to mention this above, but they are also considering doing the same for unload handlers: https://groups.google.com/a/chromium.org/g/blink-dev/c/oU1yt5cdGH8/m/3TSBIrvSBgAJ
It's at an earlier stage so maybe too early to mention in this article?